e2e_tests/conftest.py

```diff
@@ -77,3 +77,28 @@ def browser_context_args(browser_context_args):
         "ignore_https_errors": True,
         "java_script_enabled": False,
     }
+
+
+class Utils:
+    @staticmethod
+    def get_page_table_headers(page: Page):
+        return page.locator(
+            "th:not(.govuk-table--invisible-on-desktop)"
```

Comment:
is it worth adding the info about these helper functions being for the desktop version, not for mobile, to the names of these functions are class.

Reply:
@anthonyhashemi pushed a change for this